pkgs/numpy/numpy.yaml

```diff
@@ -30,6 +30,29 @@ build_stages:
       export LAPACK=None
       export BLAS=None
 
+  - when: machine == 'SGIICEX'
+    name: create-site.cfg
+    after: prologue
+    before: install
+    handler: bash
+    bash: |
+      cat > site.cfg << EOF
+      [mkl]
+      library_dirs = /app/intel/compilers/14/mkl/lib/intel64
```

Comment:
I think that this should not be in here. Rather, this should be specified in the user profile for this particular machine.

I am ok with merging this as a temporary measure, but this is not the way to go in the long term.